Frontend/src/components/librarian/LoanCreation.tsx

```diff
@@ -0,0 +1,163 @@
+import React,{useState} from "react";
```

Comment:
Remember that you should run `npx prettier --write .` to get all of your code formatted (make sure that you are in the `/Frontend` directory when you run it).

You may be able to get prettier automatically running in vscode but I have the extension and I don't think it's doing anything for me so I just run the command manually.